package.json

```diff
@@ -83,13 +83,14 @@
     }
   },
   "scripts": {
+    "all": "run-s test",
```

Comment:
Why would `all` not also `build`/ `compile`?

Reply:
It's doing it in [`pretest`](https://github.com/bpmn-io/vs-code-bpmn-io/blob/master/package.json#L92). I think this comes from the extension boilerplate we were using once we created it. Probably the VSCode environment uses this `pretest` task.

Reply:
ah - I see